html-css/week1/yasi-portfoilo/index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="description" content="Yasaman Ayazi | Student at HackYourFuture" />
+  <meta name="keywords" content="HTML, CSS, JavaScript, Web Development, Front-End, Software Engineer" />
+  <meta name="author" content="Yasaman Ayazi" />
+  <link rel="stylesheet" href="./resources/stylesheets/styles.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@300..700&family=Mulish:ital,wght@0,200..1000;1,200..1000&display=swap"
+    rel="stylesheet" />
+  <link rel="icon" type="image/x-icon" href="./resources/favicon/favicon.ico" />
+  <title>Yasaman Ayazi | Front-End Developer in Training</title>
+</head>
+
+<body>
+  <header class="head" id="landing">
+    <nav class="" id="nav-top">
+      <div class="nav-wrapper">
+        <ul class="nav-list">
+          <li class="list-item page-nav">Navigation</li>
+          <li class="list-item social-nav">Social</li>
+        </ul>
+      </div>
+    </nav>
+    <div class="head-wrapper">
+      <section class="container hero">
+        <canvas class="" id="canvas"></canvas>
+        <div class="hero-content mulish-reg">
+          <h4>
+            Hi, I'm Yasaman Ayazi. A motivated learner diving into front-end development to create dynamic user
+            experiences.
+          </h4>
+          <div class="typing">
+            <h1>Front-End Web <span>Developer</span> in Training</h1>
+            <span class="rounded"></span>
+          </div>
+          <div>
+            <a href="#contact"><button class="contact-btn mulish-reg">Contact Me</button></a>
```

Comment:
Both `<a>` and `<button>` are clickable interactive elements. There might be some browser hiccups when one is nested in the other. I see you've used the `<button>` only for it's styling properties. You could instead format your markup as:
```suggestion
            <a href="#contact"  class="contact-btn mulish-reg">Contact Me</a>
```